### DISCUSSION TITLE
Using hourly timestep instead of half hour and running into issues

### DISCUSSION THREAD

**Discussion by psbecker** [NCAR/hrldas#209] [Q&A]
Hello,

I have data for a different site with hourly forcing (as opposed to the example bondville.dat with half hour forcing). I opened the namelist and changed the FORCING_TIMESTEP to be 3600 to reflect this, but for some reason with doing this, the model now thinks it needs to look for inputs named YYYYMMDDhh.LDASIN as opposed to YYYYMMDDhhmm.LDASIN, which is what I created. I get the following error from this:
` Problems finding global attribute: ISLAKE; setting to -1
wrfinput_flnm: '../HRLDAS_forcing/run/examples/single_point/hrldas_setup_single_point_moonlake.nc'
 Did not find MAPFAC_MX, only needed for iopt_run=5
 Did not find MAPFAC_MY, only needed for iopt_run=5
netcdf_flnm: '../HRLDAS_forcing/run/examples/single_point/hrldas_setup_single_point_moonlake.nc'
LDASIN TITLE attribute: Created from HRLDAS create_point_data
WARNING:  LDASIN file has a perverse version identifier
ldasin_version =        0
LDASIN MMNINLU attribute: MODIFIED_IGBP_MODIS_NOAH

layer_bottom(1:nsoil) =    0.1000   0.4000   1.0000   2.0000
layer_top(1:nsoil)    =    0.0000   0.1000   0.4000   1.0000
Soil depth =     0.100000    0.300000    0.600000    1.000000
 k, dst_centerpoint(k) =            1   5.00000007E-02
 k, dst_centerpoint(k) =            2  0.250000000
 k, dst_centerpoint(k) =            3  0.700000048
 k, dst_centerpoint(k) =            4   1.50000000

 k, src_centerpoint(k) =            1   5.00000007E-02
 k, src_centerpoint(k) =            2  0.250000000
 k, src_centerpoint(k) =            3  0.699999988
 k, src_centerpoint(k) =            4   1.50000000
(Near) match for destination layer:  Taking destination layer at  0.0500 from source layer at  0.0500
(Near) match for destination layer:  Taking destination layer at  0.2500 from source layer at  0.2500
(Near) match for destination layer:  Taking destination layer at  0.7000 from source layer at  0.7000
(Near) match for destination layer:  Taking destination layer at  1.5000 from source layer at  1.5000
 k, dst_centerpoint(k) =            1   5.00000007E-02
 k, dst_centerpoint(k) =            2  0.250000000
 k, dst_centerpoint(k) =            3  0.700000048
 k, dst_centerpoint(k) =            4   1.50000000

 k, src_centerpoint(k) =            1   5.00000007E-02
 k, src_centerpoint(k) =            2  0.250000000
 k, src_centerpoint(k) =            3  0.699999988
 k, src_centerpoint(k) =            4   1.50000000
(Near) match for destination layer:  Taking destination layer at  0.0500 from source layer at  0.0500
(Near) match for destination layer:  Taking destination layer at  0.2500 from source layer at  0.2500
(Near) match for destination layer:  Taking destination layer at  0.7000 from source layer at  0.7000
(Near) match for destination layer:  Taking destination layer at  1.5000 from source layer at  1.5000
 NTIME =        17520 KHOUR=        8760 dtbl =    1800.00000
A)  Problem opening netcdf file: '../HRLDAS_forcing/run/examples/single_point//1986100200.LDASIN_DOMAIN1'`

Any thoughts on how to fix this?

**Comment by cenlinhe** [comment]
you can manually change the line 277 in the create_point_data.f90 by removing  "nn" in the output filename: https://github.com/NCAR/hrldas/blob/master/hrldas/HRLDAS_forcing/run/examples/single_point/create_point_data.f90#L277
to:
 write(filename,'(i4,**3**i2.2,a15)') yyyy, mm, dd, hh, ".LDASIN_DOMAIN1")

**Comment by psbecker** [comment]
that worked! Thank you!
